dir.props

```diff
@@ -57,6 +57,8 @@
     <SourceBuiltAssetsDir>$(SourceBuiltBlobFeedDir)assets/</SourceBuiltAssetsDir>
     <PrebuiltPackagesPath>$(ProjectDir)prebuilt/nuget-packages/</PrebuiltPackagesPath>
     <SourceBuiltTarBallPath>$(OutputPath)</SourceBuiltTarBallPath>
+    <SourceBuiltToolsetPath>$(LocalBlobStorageRoot)Toolset/</SourceBuiltToolsetPath>
+    <SourceBuiltRuntimePath>$(LocalBlobStorageRoot)Runtime/</SourceBuiltRuntimePath>
```

Comment:
The convention I've been following is `Dir` for dirs and `Path` for file paths. It looks like it's already mixed here. 😕

The other convention I know of is `Path` for dirs and `File` for file paths. I'd be comfortable going `Dir` + `File` to be unambiguous. I'm not sure what the official MSBuild convention is. (Or what we expect Arcade to do.) It just gets really hard to read when the suffix doesn't always mean something consistent.

Reply:
OK.  I'll go with Dir for the new ones.